packages/lab/src/progress/LinearProgress/LinearProgress.css

```diff
@@ -6,28 +6,6 @@
   font-size: var(--linearProgress-fontSize);
 }
 
-/* TODO: Odyssey now has only 2 sizes */
-.saltLinearProgress-large {
-  --linearProgress-after-top: 3px;
-  --linearProgress-value-paddingLeft: 12px;
-  --linearProgress-height: 7px;
-  --linearProgress-fontSize: 16px;
-}
-
-.saltLinearProgress-medium {
-  --linearProgress-after-top: 2px;
-  --linearProgress-value-paddingLeft: 8px;
-  --linearProgress-height: 5px;
-  --linearProgress-fontSize: 14px;
-}
-
-.saltLinearProgress-small {
-  --linearProgress-after-top: 1px;
-  --linearProgress-value-paddingLeft: 4px;
-  --linearProgress-height: 3px;
-  --linearProgress-fontSize: 11px;
-}
-
 .saltLinearProgress-disabled .saltLinearProgress-bar {
```

Comment:
something is wrong with the structure (this was a bug before) as when the track is disabled, the line shows behind the progress bar
i think the solution (since opacity is reduced) is to put the opacity at 0% on the line at all times up to the px that the progress bar is filled, if that makes sense?
another option is structural change; to change the size of the line to only be the bit remaining after the progress bar. but that isnt ideal at all because its misleading (i spoke with design about this last year)

Reply:
the misleading option seems like how its done in design though! perhaps the structural change is ok as an svg?

Reply:
yeahh I noticed this, I thought the blue had lower opacity

Reply:
went with structural change so it looks like one complete circle made from two parts of a circle, same with line. can be something we discuss in v2 but for now it looks like the design